src/Artifacts.php

```diff
@@ -4,6 +4,7 @@
 
 namespace Keboola\Artifacts;
 
+use JetBrains\PhpStorm\Pure;
```

Comment:
pozustatek?

Reply:
Jo, to jsem zapomnel commitnout fix tohodle